webapp/templates/admin_upload.html

```diff
@@ -90,6 +90,10 @@ <h2>URL payload</h2>
       <label slot="label">Results URL</label>
       <input name="result_url" slot="input">
     </paper-input-container>
+    <paper-input-container>
+      <label slot="label">Screenshots URL</label>
+      <input name="screenshot_url" slot="input">
+    </paper-input-container>
```

Comment:
Add a screenshots field to the file-upload form above: 
https://github.com/web-platform-tests/wpt.fyi/pull/1202/files#diff-1d16f5f701e65fe88d9b262fba7b5eaaL45